frontend/src/router/index.ts

```diff
@@ -23,6 +23,12 @@ const routes: Array<RouteRecordRaw> = [
     component: () => import('../views/HomeView.vue'),
     meta: { title: 'Home' }
   },
+  {
+    path: '/initiatives',
+    name: RouteNames.INITIATIVES,
+    component: () => import('../views/InitiativesView.vue'),
+    meta: { title: 'Initiatives' }
```

Comment:
If this is to be an authentication-gated page, we might need that authentication thingie we usually do here?